.github/workflows/create_additional_release_tags.yaml

```diff
@@ -17,7 +17,7 @@ jobs:
     - name: Checkout code
       uses: actions/checkout@v3
       with:
-        ref: ${{ github.event.inputs.releaseTag }}
+        ref: ${{ github.event.release.tag_name || github.event.inputs.releaseTag }}
```

Comment:
Can we use the default value?


https://github.com/actions/checkout/blob/main/README.md

    # The branch, tag or SHA to checkout. When checking out the repository that
    # triggered a workflow, this defaults to the reference or SHA for that event.
    # Otherwise, uses the default branch.
    ref: ''

Reply:
Workflow_dispatch lets you specify a tag when invoking. Try that.

Reply:
So you don't even need releaseTag argument.

Reply:
> If it's removed, then the workflow definition file must exist at the tag.

So the use of `${{ github.event.release.tag_name || github.event.inputs.releaseTag }}` is fine.